src/index.html

```diff
@@ -11,12 +11,101 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo__img"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="index.html"
```

Comment:
```suggestion
              href="#apple"
```
Instead of `index.html` you can add links to some anchors on the current page

Reply:
```
<li class="nav__item">
            <a
              class="nav__link is-active"
              href="#apple"
            >
              Apple
            </a>
```

Reply:
fixed